intake/cli/server/server.py

```diff
@@ -160,6 +173,31 @@ def initialize(self, catalog, cache, auth):
         self._cache = cache
         self.auth = auth
 
+    def get(self):
+        head = self.request.headers
```

Comment:
Doc: this is for direct access to an *entry* by name for random access where the whole catalog has not first been listed (e.g., in the case of pagination).
Correct?

Reply:
Yes, correct. Docstring added. Thank you for flagging this.